topostats/default_config.yaml

```diff
@@ -1,5 +1,5 @@
 base_dir: ./ # Directory in which to search for data files
-output_dir: ./ # Directory to output results to
```

Comment:
Not sure why this is `./` its always been `./output/` on `main` but good to have it reverted. :+1:

Reply:
reverted

Reply:
This PR _does_ revert it :wink: 

If you want different output directories when developing/testing then you could use the command line options/flags or generate and customise a configuration file and invoke with that.

Although perhaps this doesn't fit with a Notebook based workflow, but if using notebooks load the config and change the value in the dictionary might be easiest, then you don't have to remember to revert changes on the feature branch. :shrug: